turtlebot3_automation:
  ros__parameters:
    mode: "slam"
    global_frame: "map"
    robot_frame: "base_link"
    auto_start: true
    waypoint_follow_delay: 2.0
    initial_pose:
      frame_id: "map"
      position:
        x: 0.0
        y: 0.0
        z: 0.0
      yaw: 0.0
    waypoints:
      - frame_id: "map"
        position:
          x: 0.0
          y: 0.0
          z: 0.0
        yaw: 0.0
      - frame_id: "map"
        position:
          x: 1.5
          y: 0.5
          z: 0.0
        yaw: 1.57
